integreat_cms/cms/templates/pages/page_revisions.html

```diff
@@ -53,27 +57,42 @@ <h1 class="heading">
-                {% if page_translation == api_revision %}
-                    ({% translate "This is the version shown in the apps." %})
-                {% elif forloop.first %}
-                    {% if api_revision %}
-                        {% with api_revision.version as version %}
-                            ({% blocktrans %}This version is <b>not</b> displayed in the apps - but version {{ version }} instead{% endblocktrans %})
-                        {% endwith %}
-                    {% else %}
-                        ({% translate "This translation is <b>not</b> shown in the apps" %})
+                 data-status="{{ page_translation.status }}"
+                 data-hix="{{ page_translation.hix_score|floatformat:2 }}">
+                <div class="flex justify-between">
+                    <span>
+                        <label class="inline-block">{% translate "Publication status" %}:</label>
+                        {{ page_translation.get_status_display }}
+                        {% if page_translation == api_revision %}
+                            ({% translate "This is the version shown in the apps." %})
+                        {% elif forloop.first %}
+                            {% if api_revision %}
+                                {% with api_revision.version as version %}
+                                    ({% blocktrans %}This version is <b>not</b> displayed in the apps - but version {{ version }} instead{% endblocktrans %})
+                                {% endwith %}
+                            {% else %}
+                                ({% translate "This translation is <b>not</b> shown in the apps" %})
+                            {% endif %}
+                        {% endif %}
+                        {% if page_translation.minor_edit %}
+                            - <i>{% translate "Minor edit" %}</i>
+                        {% endif %}
+                    </span>
+                    {% if TEXTLAB_API_ENABLED and request.region.hix_enabled and not page_translations.first.page.hix_ignore %}
+                        <span>
+                            <label class="inline-block">{% translate "HIX score" %}:</label>
+                            {% if page_translation.hix_score %}
+                                {{ page_translation.hix_score|floatformat:2 }}
+                            {% else %}
+                                -
+                            {% endif %}
+                        </span>
                     {% endif %}
-                {% endif %}
-                {% if page_translation.minor_edit %}
-                    - <i>{% translate "Minor edit" %}</i>
-                {% endif %}
-                <span class="float-right">
-                    <label class="inline-block">{% translate "Author" %}:</label>
-                    {% firstof page_translation.creator deleted_user_text %}
-                </span>
+                    <span>
```

Comment:
```suggestion
                    <span>
                        {% if page_translation.hix_enabled and not page_translation.hix_ignore and page_translation.hix_score %}
                            <span class="pr-4">
                                <label class="inline-block">{% translate "HIX score" %}:</label>
                                {{ page_translation.hix_score|floatformat:2 }}
                            </span>
                        {% endif %}
```
I think it would look better if the HIX score would also be aligned to the right and not floating in the middle.
Also, we can make use of the translation properties here:
https://github.com/digitalfabrik/integreat-cms/blob/e654925ba72fe2930e8e36acd8eabac82af1c6c7/integreat_cms/cms/models/abstract_content_translation.py#L510-L533

This also makes sure that we don't show the HIX value on languages that are not enabled for HIX.